docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -209,22 +209,24 @@ tag::expand-wildcards[]
 `expand_wildcards`::
 +
 --
-(Optional, string) Controls what kind of indices that wildcard expressions can
-expand to. Multiple values are accepted when separated by a comma, as in
-`open,hidden`. Valid values are:
+(Optional, string)
+Type of index that wildcard expressions can match. If the request can target
+data streams, this argument determines whether wildcard expressions match
+hidden data streams. Supports comma-separated values, such as `open,hidden`.
+Valid values are:
 
 `all`::
-Expand to open and closed indices, including <<hidden-indices,hidden indices>>.
+Match any data stream or index, including <<hidden-indices,hidden>> ones.
 
 `open`::
-Expand only to open indices.
+Match non-hidden data streams and open, non-hidden indices.
 
 `closed`::
-Expand only to closed indices.
+Match closed indices.
```

Comment:
Good point. I was initially side-stepping this, but it may be better to be explicit.

With 04cd44009a2f27b9f2ef93f1125d5f17edd76a13, we now note that data streams cannot be closed. I don't think we probably need that for `open`, but let me know.